src/rgw/rgw_rados.cc

```diff
@@ -6640,7 +6648,11 @@ int RGWRados::guard_reshard(BucketShard *bs,
   for (int i = 0; i < NUM_RESHARD_RETRIES; ++i) {
     r = bs->init(pobj->bucket, *pobj, nullptr /* no RGWBucketInfo */);
     if (r < 0) {
-      ldout(cct, 5) << "bs.init() returned ret=" << r << dendl;
```

Comment:
assuming this is the same as RGWRados::Bucket::UpdateIndex::guard_reshard()--but I didn't realize there were two guard_reshard() methods